src/pages/People.tsx

```diff
@@ -0,0 +1,132 @@
+import { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Loader } from '../components/Loader';
+import { getPeople } from '../api';
+import { peopleStore } from '../Store';
+
+export const People: React.FC = () => {
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(false);
+  const { human } = useParams();
+  const { people, setPeople } = peopleStore();
+
+  useEffect(() => {
+    setLoading(true);
+    getPeople()
+      .then(setPeople)
+      .catch(err => {
+        setError(true);
+        Error(err.message);
+      }).finally(() => setLoading(false));
+  }, []);
+
+  const parentLink = (name: string | null) => {
+    if (!name) {
+      return '-';
+    }
+
+    const parent = people.find(el => el.name === name);
+
+    if (!parent) {
+      return name;
+    }
+
+    return (
+      <Link
+        className={cn(
+          { 'has-text-danger': parent.sex === 'f' },
+        )}
+        to={`../${parent.slug}`}
+      >
+        {parent.name}
+      </Link>
+    );
+  };
+
+  return (
+    <main className="section">
+      <div className="container">
+        <h1 className="title">People Page</h1>
+      </div>
+      <div className="block">
+        <div className="box table-container">
+
+          {error && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              Something went wrong
+            </p>
+          )}
+
+          {people.length === 0
```

Comment:
```suggestion
          {people.length === 0 && !loading
```
this message shouldn't be visible during loading:
![image](https://github.com/mate-academy/react_people-table-basics/assets/86923089/e7391660-758a-4d5e-87be-18cebbe5f8df)